.github/workflows/postsubmit.yml

```diff
@@ -63,3 +63,41 @@ jobs:
           publish_branch: release-2.4
           keep_files: true
           destination_dir: clusterImageSets
+
+  sync-clusterimageset-release-2_5:
+    name: sync-clusterimageset-release-2.5
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: sync-clusterimageset
+        run: make sync-images-job
+        env:
+          TARGET_BRANCH: release-2.5
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          publish_dir: ./clusterImageSets
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: release-2.5
+          keep_files: true
+          destination_dir: clusterImageSets
+
+  sync-clusterimageset-main:
```

Comment:
why we need to update main. I think we should not push to main?

Reply:
> why we need to update main. I think we should not push to main?

The 2.3 user still use the main branch, So we also need to sync the clusterimagesets in main branch.

Reply:
I think we already update console chart to sub on 2.3 branch. For main branch. I'd rather we keep it static for backward compatible.